app/lib/dags/experimental.json.erb

```diff
@@ -113,7 +113,7 @@
       "class": "PipelineStepGenerateCoverageViz",
       "module": "idseq_dag.steps.generate_coverage_viz",
       "additional_files": {
-        "info_db": "s3://idseq-database/alignment_data/2018-12-01/nt_info.sqlite3"
+        "info_db": "<%= @attribute_dict[:nt_info_db] %>"
```

Comment:
just merged mine, so in the jbuilder you can just change to `info_db: attr[:nt_loc_db]` as well